mint.json

```diff
@@ -386,7 +386,8 @@
       "group": "Additional Resources",
       "pages": [
         "additional-resources/rate-limits",
-        "additional-resources/glossary"
+        "additional-resources/glossary",
+        "additional-resources/idempotency"
```

Comment:
Closer grouping of Rate Limits and Idempotency as they are related with retry behaviour.

```suggestion
        "additional-resources/idempotency",
        "additional-resources/glossary"
```